src/clj/game/cards/programs.clj

```diff
@@ -598,20 +598,20 @@
     :abilities [{:cost [:credit 1] :msg (msg "break 1 " (:subtype-target card) " subroutine")}]}
 
    "Chisel"
-   {:effect (effect (system-say "Corp, please press No Action to trigger Chisel"))
+   {:implementation "Encounter effect is manual."
     :hosting {:req (every-pred ice? can-host?)}
+    :abilities [{:req (req (= (:cid current-ice) (:cid (:host card))))
```

Comment:
There is a shortcut for this - `same-card?` or something

Reply:
Good call. Probably should change all the other cards like this too then, lol.